app/models/assessment_detail.rb

```diff
@@ -71,6 +71,8 @@ def categories_for(application_type)
         category_keys - %w[publicity_summary amenity]
       when :prior_approval
         category_keys - %w[consultation_summary]
+      when :planning_permission
+        category_keys - %w[amenity]
```

Comment:
Shouldn't this be driven by this field on the assessment type?

https://github.com/unboxed/bops/blob/fa908c3cf0560d5dfd9c618c3673eb0b9b885aeb/db/schema.rb#L80

Reply:
Added by Celia a couple of weeks ago:
https://github.com/unboxed/bops/commit/5599c399df820da7356bf1eab227ca249652fbe6

Reply:
Good point! I remember that being added but I assumed without double-checking that it was for something different. So presumably these lists were actually duplicated previously, and this one didn't get updated. Will fix now.